src/frontend/magnify/apps/magnify-site/public/config.json

```diff
@@ -4,5 +4,6 @@
   "KEYCLOAK_CLIENT_ID": "magnify-front",
   "KEYCLOAK_EXPIRATION_SECONDS": 1800,
   "KEYCLOAK_REALM": "magnify",
+  "MAGNIFY_SHOW_REGISTER_LINK": false,
```

Comment:
Should be set to `true` for dev purpose nope ?

Reply:
Yes